core/modules/catalog/helpers/variant/checkFullConfigurationMatch.ts

```diff
@@ -0,0 +1,25 @@
+import toString from 'lodash/toString'
+import omit from 'lodash/omit'
+
+/**
+ * Checks if full coniguration matches for specific variant
+ */
+export default function checkFullConfigurationMatch (configuration, variant): boolean {
+  if (!variant || !configuration) return false
+  const configProperties = Object.keys(omit(configuration, ['price']))
```

Comment:
why you make omit for price? 🤔 just curious, can price be set as configuration property?